cluster/examples/README.md

```diff
@@ -0,0 +1,17 @@
+# Crossplane Examples
+
+In this directory you will find example workloads demonstrating the usage of various resources supported by Crossplane for each of the supported Clouds.
+
+[Cloud provider credentials](../../docs/cloud-providers.md) will need be needed be before using any of these examples.
+
+| Directory | Description | AWS | Azure | GCP
+| ---       | ---         | --- | ---   | ---
+| [cache](cache/)     | Provisions a Managed Redis service from any cloud. | Redis | Redis | CloudMemoryStore Redis |
+| [compute](compute/)   | Creates a WordPress Workload that runs in a Crossplane created Kubernetes cluster using a Crossplane created managed MySQL service. ([Using Compute Workloads](https://github.com/crossplaneio/crossplane/blob/master/design/complex-workloads.md#complex-workloads-in-crossplane), [Legacy](https://github.com/crossplaneio/crossplane/issues/456)) | [EKS + RDS](../../docs/workloads/aws/wordpress-aws.md) | [AKS + Azure SQL](../../docs/workloads/azure/wordpress-azure.md) | GKE + Cloud SQL |
```

Comment:
we are only linking to a handful of provider specific examples.  can we be consistent here? either link to them all, or not have any links?

Reply:
The GCP provider does not have documentation for the `compute` example, since the documentation was (partially) updated for the `workloads` example.

I can go either way on the GCP wordpress example. Revise the documentation to refer to use the `compute` example revise it for the `complex` example.  As-is, I have found a few more errors in the documentation that I will update in a follow-up commit to this branch.

While the GCP `compute` example exists, there is no documentation link for it.  I should lead that table with a note that the links refer to documentation for that example, if it exists.  Or alternatively, each link could refer to the directory with the example in it and documentation links could be a secondary "docs" link in the table cell.

Reply:
I see, I think I mixed up in my head the examples links and the docs links, which are 2 separate things.  Everything in this table has an examples link, but only some of them have actual written documentation.  Makes sense that not all of them have docs links then.